chains/dreyerx/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiKey: abc
+    apiUrl: https://scan.dreyerx.com/api
+    family: blockscout
+    name: DreyerX Scan
+    url: https://scan.dreyerx.com
+chainId: 23451
+displayName: DreyerX
+domainId: 23451
+isTestnet: false
+name: dreyerx
+nativeToken:
+  decimals: 18
+  name: DreyerX
+  symbol: DRX
+deployer:
+  name: billalxcode
+  url: https://github.com/billalxcode
```

Comment:
this will fail linter, please run `yarn lint:fix` and `yarn prettier`

Reply:
i've fixed